stories/multi-select.stories.js

```diff
@@ -120,4 +121,33 @@ storiesOf("React Multi Select", module)
         showSelectAll={boolean("Show select all", true)}
       />
     );
-  });
+  })
+    .add("display custom selected item", () => {
+        const items = Array.apply(null, {length: 10}).map((i, index) => ({
+            id: index,
+            value: `item ${index}`,
+            label: <div><Icon>star</Icon>{`Item ${index}`}</div>
+        }));
+
+        const selectedItemFunc = () => <div>selected</div>;
+
+        class SearchComponent extends Component {
+            render() {
+                return (
+                    <div>custom search <Input value={this.props.value} onChange={this.props.onChange}/></div>
```

Comment:
This can be stateless, I also suggest not using the material component to make it clearer